elm.json

```diff
@@ -1,12 +1,13 @@
 {
     "type": "package",
-    "name": "cuducos/elm-format-number",
+    "name": "coinop-logan/elm-format-number",
```

Comment:
This change should not be included in the PR, as this package is still in my account. Please, revert it.